models/migrations/migrations.go

```diff
@@ -406,6 +406,8 @@ var migrations = []Migration{
 	NewMigration("Drop old CredentialID column", dropOldCredentialIDColumn),
 	// v223 -> v224
 	NewMigration("Rename CredentialIDBytes column to CredentialID", renameCredentialIDBytes),
+	// v224 -> v225
```

Comment:
you canot backport migrations!

Reply:
To explain, if we introduce migration 225 on 1.17, the same migration number would be skipped when upgrading to 1.18. And even if we leave space in the ids, I think it's still not save as one migration could theoretically depend on another.

Reply:
Makes sense, thank you!